task-header.html

```diff
@@ -0,0 +1,71 @@
+<html>
+	<head>
+		<link rel="stylesheet" href="task-header.css">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0"> 
+	</head>
+	
+	<body>
+		<nav class="firstnav">
+			<div class="infobar">
+				<a href="aboutme"> About Me </a>
+				<a href="portfolio"> Portfolio </a>
+				<a href="contactme"> Contact Me </a>
+			</div>
+			
+			<div class="social">
+				<a href="https://uk.linkedin.com/"><img src="LinkedinLogo.png" img class="socialicons"></a>
+				<a href="https://twitter.com/?lang=en-gb"><img src="twitter.png" img class="socialicons"></a>
+				<a href="https://www.instagram.com/"><img src="instagram.png" img class="socialicons"></a>
```

Comment:
indenting img might be easier?
```suggestion
				<a href="https://uk.linkedin.com/">
				             <img src="LinkedinLogo.png" img class="socialicons">
				</a>
				<a href="https://twitter.com/?lang=en-gb">
				         <img src="twitter.png" img class="socialicons">
				</a>
				<a href="https://www.instagram.com/">
				            <img src="instagram.png" img class="socialicons">
				</a>
```

Reply:
Yep, agree with that.